projects/step-frontend/src/lib/modules/execution/services/report-tree-node-utils.service.ts

```diff
@@ -10,9 +10,10 @@ export class ReportTreeNodeUtilsService implements TreeNodeUtilsService<ReportNo
   private readonly hasChildrenFlags: Record<string, boolean> = {};
 
   constructor(
-    @Inject(EXECUTION_TREE_PAGING) private _paging: ExecutionTreePaging,
+    @Inject(EXECUTION_TREE_PAGING) private _paging: ExecutionTreePagingSetting,
     private _artefactTypes: ArtefactService,
-    private _controllerService: ControllerService
+    private _controllerService: ControllerService,
+    private _executionTreePaging: ExecutionTreePaging
   ) {}
```

Comment:
Make sense to replace the constructor with fields, initialized by `inject` function

Reply:
As this is a fix for a minor release I want to touch least amount of code as possible